sim/dex.ts

```diff
@@ -363,15 +363,25 @@ export class ModdedDex {
 		}
 		if (!this.data.Pokedex.hasOwnProperty(id)) {
 			let aliasTo = '';
-			if (id.startsWith('mega') && this.data.Pokedex[id.slice(4) + 'mega']) {
-				aliasTo = id.slice(4) + 'mega';
-			} else if (id.startsWith('m') && this.data.Pokedex[id.slice(1) + 'mega']) {
-				aliasTo = id.slice(1) + 'mega';
-			} else if (id.startsWith('primal') && this.data.Pokedex[id.slice(6) + 'primal']) {
-				aliasTo = id.slice(6) + 'primal';
-			} else if (id.startsWith('p') && this.data.Pokedex[id.slice(1) + 'primal']) {
-				aliasTo = id.slice(1) + 'primal';
-			}
+			const formeNames = {
+				a: 'alola',
+				alolan: 'alola',
+				g: 'galar',
+				galarian: 'galar',
+				gigantamax: 'gmax',
+				gmax: 'gmax',
+				m: 'mega',
+				mega: 'mega',
+				p: 'primal',
+				primal: 'primal',
+			};
+			Object.keys(formeNames).forEach(forme => {
```

Comment:
theres no reason to use forEach

Reply:
i thought it was simpler, how would you do it

Reply:
`for (const formeName of Object.keys(formeNames))` or just using `for (const i in formeNames)`